server/models/categoryModel.js

```diff
@@ -21,6 +21,10 @@ Category.init(
       type: DataTypes.STRING,
       allowNull: false,
     },
+    type: {
+      type: DataTypes.ENUM('income', 'expense'),
+      allowNull: false,
```

Comment:
```
module.exports = {
 
  CategoryType: Object.freeze({
    INCOME: 'I',
    EXPENSE: 'E',
  }),
};

```

and import

Reply:
Changed in commit b65f80fff5815e341e0b7113865a636599e181a1